internal/net/call/msg.go

```diff
@@ -49,31 +49,31 @@ const currentVersion = initialVersion
 // # Message formats
 //
 // All messages have the following format:
-//	id	[8]byte
-//	type	[1]byte		-- messageType
-//	length	[7]byte		-- length of the remainder of the message
-//	payload	[length]byte	-- message-type-specific data
+//    id        [8]byte       -- identifier used to track the message
+//    type      [1]byte       -- messageType
+//    length    [7]byte       -- length of the remainder of the message
+//    payload   [length]byte  -- message-type-specific data
```

Comment:
There were some mixes of tabs and spaces in this comment that caused it to format poorly when tabs were set to 4 spaces. I switched them to spaces so that it formats correctly regardless of tab width.